packages/core/src/render3/error_code.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export const enum RuntimeErrorCode {
+  // Internal Errors
+
+  // Change Detection Errors
+  EXPRESSION_CHANGED_AFTER_CHECKED = '100',
+
+  // Dependency Injection Errors
+  INSTANTIATE_CYCLIC_DEPENDENCY = '200',
```

Comment:
I think we should add the "DI" into the error, otherwise this error code sounds too generic (there might be cyclic deps in imports, etc):
```suggestion
  CYCLIC_DI_DEPENDENCY = '200',
```